Makefile

```diff
@@ -140,7 +140,7 @@ test: ## Run unit tests
 
 .PHONY: test-e2e
 test-e2e: ## Run e2e tests
-	hack/e2e.sh $(KUBECTL)
+	hack/e2e.sh ${KUBECTL}
```

Comment:
I think this change is wrong. For a Makefile, I think we want parens, not braces.

Reply:
nevermind, either works in POSIX and GNU make